tests/Fixer/LanguageConstruct/IsNullFixerTest.php

```diff
@@ -206,4 +195,37 @@ public function provideYodaFixCases()
             ),
         );
     }
+
+    /**
+     * @dataProvider provideNonYodaFixCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testNonYodaFix($expected, $input)
+    {
+        $this->fixer->configure(array('use_yoda_style' => false));
+        $this->doTest($expected, $input);
+    }
+
+    public function provideNonYodaFixCases()
+    {
+        return array(
+            array(
+                '<?php $x = $y === null;', '<?php $x = is_null($y);',
+            ),
+            array(
+                '<?php $b = a(a(a(b() === null) === null) === null) === null;',
```

Comment:
Wow, this code is beautiful. Thanks for the idea. I think I will start writing all my projects like this. ;)

Reply:
this test is to check if `is_null` inside `is_null` (multiple times) works properly or fixer jump over some of them.
as we don't recommend writing code like that, we don't want to crash if Fixer visit it.

Reply:
Yeah I know it's a test, I was kidding. It's good to have that test. I just thought it was the ugliest code style I have ever seen. ;-) If someone does that, then we can really talk about bad coding haha.